<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="ProfileAssignmentsBusiness">
  <testsuite name="Login to CB as primaryadmin" id="4c7c9f81-9cb3-4fa7-bb41-78212e58b636" tests="1" time="1063">
    <testcase name="status code is 201 : Created"/>
  </testsuite>
  <testsuite name="Add Business Profile" id="f26269b8-5d05-4fcc-a5db-ae54f06d71b3" tests="0" time="0">
    <error>
      <![CDATA[Iteration: 0
RequestError: Error: read ECONNRESET
    at exports._errnoException (util.js:907:11)
    at TLSWrap.onread (net.js:557:26)

---
testScriptError: ReferenceError: responseBody is not defined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:3:27)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
  </testsuite>
  <testsuite name="get profile by Id copy" id="a46dffcc-3e9d-457d-8bd9-1f1b2a4c881d" tests="1" time="830">
    <error>
      <![CDATA[Iteration: 0

---
testScriptError: TypeError: Cannot read property 'ChannelProfiles' of undefined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:5:21)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
    <testcase name="status code is 200 : OK">
      <failure type="AssertionFailure">
        <![CDATA[Failed 1 times.]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="business user add with profile" id="35c1f08d-f234-4917-8e62-494e92aeee19" tests="1" time="704">
    <error>
      <![CDATA[Iteration: 0

---
testScriptError: TypeError: Cannot read property 'Id' of undefined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:5:51)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
    <testcase name="status code is 201 : Created">
      <failure type="AssertionFailure">
        <![CDATA[Failed 1 times.]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="Get Business Users with profile" id="2f8c46d2-fe55-42d0-9503-916dba9ec2b0" tests="0" time="663">
    <error>
      <![CDATA[Iteration: 0

---
testScriptError: TypeError: Cannot read property 'UserName' of undefined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:4:53)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
  </testsuite>
  <testsuite name="modify business user" id="a6e873c4-42fd-417a-bd6c-9b5dfc3fba14" tests="1" time="1957">
    <testcase name="status code is 204 : No Content">
      <failure type="AssertionFailure">
        <![CDATA[Failed 1 times.]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="Get Business Users with profile copy" id="71d4cc4b-1a30-44a6-86bc-2affcd03db7b" tests="0" time="1221">
    <error>
      <![CDATA[Iteration: 0

---
testScriptError: TypeError: Cannot read property 'UserName' of undefined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:4:53)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
  </testsuite>
  <testsuite name="business user add without profile" id="6aacbbaa-2bee-447f-a516-a5f1d9d879ee" tests="1" time="674">
    <testcase name="Profile is required."/>
  </testsuite>
  <testsuite name="business user add with channel profile" id="ea3dc15c-8ae3-4a0e-b721-edbe60623e74" tests="1" time="676">
    <testcase name="Profile ID is Invalid.">
      <failure type="AssertionFailure">
        <![CDATA[Failed 1 times.]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="business user add with retailUser profile" id="3f2a898f-3d85-4b9d-b7b6-c234a0571cc8" tests="1" time="2079">
    <testcase name="Profile ID is Invalid."/>
  </testsuite>
  <testsuite name="business user add with invalid profileId" id="fd2b9772-59ff-4c8f-9be2-40b444522a04" tests="1" time="756">
    <testcase name="Profile ID is Invalid."/>
  </testsuite>
  <testsuite name="business user add with multiple profile" id="9bde90b7-e028-4718-942a-584d1829cb40" tests="1" time="661">
    <testcase name="User can be assigned to only one Profile."/>
  </testsuite>
  <testsuite name="Login to CB as jolina" id="2cf96c55-a81e-40e6-949e-1da8391dc6d7" tests="1" time="784">
    <testcase name="status code is 201 : Created"/>
  </testsuite>
  <testsuite name="Add Business Profile with user not entitled to add" id="181748ed-9771-4544-ba29-83856a294b2d" tests="1" time="272">
    <testcase name="status code is 400 : Bad Request"/>
  </testsuite>
  <testsuite name="business user add with profile with user not entitled" id="b0c2bddb-8711-4ad2-a2a0-d2e322af615c" tests="1" time="648">
    <testcase name="status code is 400 : Bad Request"/>
  </testsuite>
  <testsuite name="Login to CB as secondaryadmin" id="85c8a096-d447-4787-b7a7-e31ffc587d11" tests="1" time="1255">
    <testcase name="status code is 201 : Created"/>
  </testsuite>
  <testsuite name="Add Business Profile with secondaryadmin" id="36b30a8a-d795-4ecb-9532-bfbd4677dbcb" tests="1" time="284">
    <testcase name="status code is 201 : Created"/>
  </testsuite>
  <testsuite name="business user add with secondaryadmin" id="56dcfbad-823f-41e8-8e68-c0f28ea76543" tests="1" time="2191">
    <testcase name="status code is 201 : Created"/>
  </testsuite>
  <testsuite name="delete assigned profile" id="0d0a7938-3609-4282-bbbe-0e3631e604d3" tests="0" time="0">
    <error>
      <![CDATA[Iteration: 0
RequestError: Error: read ECONNRESET
    at exports._errnoException (util.js:907:11)
    at TLSWrap.onread (net.js:557:26)

---
testScriptError: ReferenceError: responseBody is not defined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:3:45)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
  </testsuite>
</testsuites>